test/base.test.js

```diff
@@ -5,29 +5,13 @@ var Airtable = require('../lib/airtable');
 
 jest.mock('request', () => jest.fn((options, cb) => {
     cb(null, {
-        user: {
-            username: options.body && options.body.username
-        },
-        token: 'tokXyz'
+        body: {
```

Comment:
Have you validated this response body shape against the result in #1? I don't think this is how it looks, check it out here: https://github.com/calblueprint/airlock/blob/master/server/controllers/authController.js#L45-L48